src/apps/AdminApp/AdminRoot.js

```diff
@@ -812,23 +813,24 @@ class AdminRoot extends Component {
           )
       });
 
-      let users_grid = feeds.map((feed,i) => {
-          if(feed) {
-              //let qt = users[feed.display.id].question;
-              //let st = users[feed.display.id].sound_test;
-              let feed_user = users.find(u => feed.id === u.rfid);
-              let qt = feed_user && !!feed_user.question;
-              return (
-                  <Table.Row active={feed.id === this.state.feed_id} key={i} onClick={() => this.getUserInfo(feed_user)} >
-                      <Table.Cell width={10}>{qt ? q : ""}{feed.display.display}</Table.Cell>
-                      {/*<Table.Cell positive={st} width={1}>{st ? v : ""}</Table.Cell>*/}
-                      <Table.Cell width={1}></Table.Cell>
-                  </Table.Row>
-              )
+      const users_grid = feeds.map((feed,i) => {
+          if (!feed) {
+            return undefined;
```

Comment:
undefined is mostly used by js runtime. we at userland use null.

Reply:
Done.